src/ExtranetBundle/objects/WebModules/MTExtranetCore/MTExtranetCoreEndPoint.class.php

```diff
@@ -201,7 +201,7 @@ public function Register($sSuccessURL = null, $sFailureURL = null)
             $this->PrepareSubmittedData($aData);
             $extranetUserProvider->reset();
             $oUser = $extranetUserProvider->getActiveUser();
-            $oUser->LoadFromRow($aData);
+            $oUser->LoadFromRow($aData, true);
```

Comment:
This may have unexpected consequences for existing installations. For example, if the input field is of type `date`, then the date will arrive in iso format (yyyy-mm-dd). I would expect the `LoadFromRow` to break this, if `true` is passed.

`PrepareSubmittedData` may be a better hook to change the date input format if required - or, alternatively the frontend input could be changed to type date.